e2e-tests/oauth-flow/rfc021/node-A/nginx.conf

```diff
@@ -43,5 +44,18 @@ http {
         proxy_set_header X-Ssl-Client-Cert $ssl_client_escaped_cert;
         proxy_pass http://nodeA-backend;
       }
+
+      location /resource {
+          js_content oauth2.introspectAccessToken;
+      }
+
+      # Location in javascript subrequest.
+      # this is needed to set headers and method
+	  location /_oauth2_send_request {
```

Comment:
A year from now on you'll read this and think; who came up with this...

Reply:
I'll add my name.... I mean the link to the blog post.